docs/source/tutorial_ph.md

```diff
@@ -0,0 +1,246 @@
+while preserving the port-Hamiltonian structure:
+
+- pH-IRKA ({class}`~pymor.reductors.ph.ph_irka.PHIRKAReductor`),
+- PRBT ({class}`~pymor.reductors.bt.PRBTReductor`),
+- passivity preserving model reduction via spectral factorization
+  ({class}`~pymor.reductors.spectral_factor.SpectralFactorReductor`).
+
+In this section, we apply all three reductors on our toy example and compare
+their performance. All three reductors are described in {cite}`BU22` in more detail.
+
+Note: Currently, the {class}`~pymor.reductors.bt.PRBTReductor` and
+{class}`~pymor.reductors.spectral_factor.SpectralFactorReductor` reductors require
+{math}`D` to be nonsingular. The MSD example has a zero {math}`D` matrix. Therefore,
+we have to add a small regularization feedthrough term. This is a limitation of the
+current implementation, since in the background the KYP-LMI is converted
+into a Riccati equation, which is only possible for nonsingular {math}`D`.
+For {func}`~pymor.models.iosys.PHLTIModel.from_passive_LTIModel`, {math}`D` must be
+nonsingular for the same reasons.
+
+```{code-cell}
+import numpy as np
+from pymor.models.iosys import PHLTIModel
+from pymor.models.examples import msd_example
+
+J, R, G, P, S, N, E, Q = msd_example(50, 2)
+
+# tolerance for solving the Riccati equation instead of KYP-LMI
+# by introducing a regularization feedthrough term D
+# (required for PRBTReductor and SpectralFactorReductor reductors)
+S += np.eye(S.shape[0]) * 1e-12
+
+fom = PHLTIModel.from_matrices(J, R, G, S=S, Q=Q, solver_options={'ricc_pos_lrcf': 'scipy'})
+```
+
+The `ricc_pos_lrcf` solver option refers to the solver used for the underlying
+Riccati equation relevant for {class}`~pymor.reductors.bt.PRBTReductor` and
+{class}`~pymor.reductors.spectral_factor.SpectralFactorReductor`. Possible choices are
+`scipy`, `slycot` or `pymess` (if installed).
+
+### pH-IRKA
+
+The pH-IRKA reductor {class}`~pymor.reductors.ph.ph_irka.PHIRKAReductor` directly returns
+a ROM of type {{ PHLTIModel }}. pH-IRKA works similar to the standard IRKA reductor
+{class}`~pymor.reductors.h2.IRKAReductor`, but with fewer degrees of freedom to preserve
+the port-Hamiltonian structure.
+
+```{code-cell}
+from pymor.reductors.ph.ph_irka import PHIRKAReductor
+
+reductor = PHIRKAReductor(fom)
+rom1 = reductor.reduce(10)
+print(f'rom1 is of type {type(rom1)}.')
+```
+
+### Positive-real balanced truncation (PRBT)
+
+Positive-real balanced truncation (PRBT) works similar to the general balanced truncation
+method described in {doc}`tutorial_bt`, but uses positive real controllability
+and observability Gramians instead. PRBT preserves passivity, but returns a ROM
+of type {{ LTIModel }}. Thus, we convert the ROM into a {{ PHLTIModel }} in a
+post-processing step. PRBT can be used with any passive {{ LTIModel}} FOM.
+
+```{code-cell}
+from pymor.reductors.bt import PRBTReductor
+
+reductor = PRBTReductor(fom)
+rom2 = reductor.reduce(10)
+rom2 = PHLTIModel.from_passive_LTIModel(rom2)
```

Comment:
```suggestion
rom2 = reductor.reduce(10)
rom2 = rom2.with_(solver_options={'ricc_pos_lrcf': 'slycot'})
rom2 = PHLTIModel.from_passive_LTIModel(rom2)
```

Reply:
The solver from `pymess` raises `NotImplementedError` (default when `pymess` is installed, as is the case for the docs building on CI) and `scipy` raises `LinAlgError: The associated Hamiltonian pencil has eigenvalues too close to the imaginary axis`. The `slycot` solver seems to work fine.